client/components/TestQueueRow/index.jsx

```diff
@@ -495,23 +491,7 @@ const TestQueueRow = ({
                     </div>
                     {isSignedIn && isTester && (
                         <div className="secondary-actions">
-                            {isAdmin &&
-                                !isLoading &&
-                                !testPlanReport.conflictsLength && (
```

Comment:
This part of the condition (concerning the existence of conflicts) isn't present in the refactored rendering code. Is that intentional?

Reply:
Wow thanks so much for catching this!